ui/pages/Project/components/ProjectPageUI.jsx

```diff
@@ -79,79 +76,64 @@ const NO_DETAIL_FIELDS = [
   { id: FAMILY_FIELD_DESCRIPTION, colWidth: 6 },
 ]
 
-const ProjectPageUI = React.memo((props) => {
-  const exportUrls = [
-    { name: 'Families', data: props.familyExportConfig },
-    { name: 'Individuals', data: props.individualsExportConfig },
-    { name: 'Samples', data: props.samplesExportConfig },
-  ]
-
```

Comment:
this component did not change except `exportUrls` is a passed in props now instead of being computed here